zivid_camera/test/test_zivid_camera.cpp

```diff
@@ -327,39 +459,41 @@ TEST_F(ZividNodeTest, testCaptureCameraInfo)
   auto depth_camera_info_sub =
       subscribe<sensor_msgs::CameraInfo>(depth_camera_info_topic_name, [&](const auto& r) { depth_camera_info = *r; });
 
-  enableFirst3DFrame();
-  zivid_camera::Capture capture;
-  ASSERT_TRUE(ros::service::call(capture_service_name, capture));
-  short_wait_duration.sleep();
+  std::optional<sensor_msgs::CameraInfo> snr_camera_info;
```

Comment:
Looks like we have approx the same code three times in this test. Perhaps we can re-use in a function?